static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -145,7 +149,12 @@ export default function GroupSidebar({
           />
         </SidebarSection.Title>
         <SidebarSection.Content>
-          <StyledAvatarList users={participants} avatarSize={28} maxVisibleAvatars={13} />
+          <StyledAvatarList
+            users={userParticipants as AvatarUser[]}
```

Comment:
We want to avoid casting as much as possible. The problem here is that TS isn't great at discriminating arrays with `.filter()` so you need to help it above with a type guard ([docs here](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards)):

```
const userParticipants = participants.filter((p): p is UserParticipant => p.type === 'user');
```